docs/.vitepress/navbar.ts

```diff
@@ -0,0 +1,46 @@
+import { DefaultTheme } from 'vitepress';
+
+const navbar: DefaultTheme.NavItem[] = [
+    {
+        text: 'Tutorials',
+        items: [
+            //
+            { text: 'Basics', link: '/tutorials/general/basics/index' },
+            { text: 'Smart Contracts', link: '/tutorials/smart-contracts/index' },
+            { text: 'Uniq Factories', link: '/tutorials/uniq-factories/index' },
+            { text: 'Token Swap', link: '/tutorials/token-swap/index' },
+        ],
+    },
+    {
+        text: 'Game Developers',
+        items: [
+            //
+            { text: 'Overview', link: '/game-developers/index' },
+            { text: 'Game Developer Center', link: '/game-developers/game-dev-center/index' },
+            { text: 'Unity', link: '/game-developers/unity/index' },
+            { text: 'Unreal', link: '/game-developers/unreal/index' },
+            { text: 'Web', link: '/game-developers/web/index' },
+        ],
+    },
+    {
+        text: 'Products',
+        items: [
+            //
+            { text: 'Quick Links', link: '/products/index' },
```

Comment:
Kind of strange that you need to go to `Quick links` to get the full list of products. Maybe something like an `Overview` would fit better?

Reply:
Overview conveys, there is some short description of the matter while quick links show exactly what we want to show to the user there.